src/lib.rs

```diff
@@ -11,12 +11,13 @@ use tracing::Level;
 
 pub const TEMPLATE_PATH: &str = "app/templates";
 
-lazy_static! {
-    pub static ref TEMPLATES: Environment<'static> = {
+fn get_reloader() -> AutoReloader {
+    AutoReloader::new(|notifier| {
         let mut env = Environment::new();
         env.set_loader(path_loader(TEMPLATE_PATH));
-        env
-    };
+        notifier.watch_path(TEMPLATE_PATH, true);
+        Ok(env)
+    })
```

Comment:
this thing allows us to change htmx templates inside of `/templates` and see the updated html without having to restart the rust server